lms/static/scripts/frontend_apps/components/ErrorDisplay.js

```diff
@@ -26,7 +26,7 @@ function toSentence(str) {
  *
  * @typedef ErrorLike
  * @prop {string} [message]
- * @prop {Object} [details] - Optional JSON-serializable details of the error
+ * @prop {object} [details] - Optional JSON-serializable details of the error
```

Comment:
From reading the code below it looks to me like this should be `object|string`.

Reply:
👍🏼